index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fileReader = require('./lib/file-reader.js');
+
+module.exports = exports = {};
+
+exports.files = ['one.txt', 'two.txt', 'three.txt'];
+
+exports.printFiles = function(file, callback) {
+  fileReader.readFile(file, function(err, data) {
+    if(err) return callback(err);
+    console.log(`${file}: ${data}`);
+  });
+};
+
+for (var i = 0; i < exports.files.length; i++) {
+  exports.printFiles(`${__dirname}/data/${exports.files[i]}`);
+}
```

Comment:
Way to use an array and loop through the array to read each file, a nice alternative to explicitly invoking fileReader 3 times!